docs/csharp/quick-starts/list-collection.yml

```diff
@@ -17,6 +17,10 @@ items:
   content: |
     Run the following code in the interactive window. To do that, type the following code block in the interactive window (replace "<name>" with your name) and click the **Run** button:
 
+    > [!NOTE]
+    > If you're starting from the code you wrote in [dot.net](https://dot.net/), this step is repeated. Copy and paste the code, and then move on to the next steps.
+
```

Comment:
Maybe this is better? dot.net is just a redirect, so we should point to the site URL. There will be a more specific URL once the new content is published.

> If you're coming from the [.NET In-Browser Tutorial](https://microsoft.com/net), you've already completed this step. Copy and paste the following code into the interactive window, then move on to [the next step](<give them an easy link>).